fe/src/main/java/org/apache/doris/plugin/PluginContext.java

```diff
@@ -17,10 +17,19 @@
 
 package org.apache.doris.plugin;
 
+import org.apache.doris.service.FrontendOptions;
+
 public class PluginContext {
 
     // the dir path where the plugin's files saved
     private String pluginPath;
+    // the identity of the FE which load this plugin. Some plugins need to know which FE is running with it.
+    // currently, we just use FE ip
+    private String feIdentity;
```

Comment:
How to handle one machine，multi Doris FE instances. Do we need to add FE port to feIdentity?

Reply:
Currently we do not support multi FE with same IP...
But add a port is not a big deal. I will add it.